Bottom: fe2673b7d75e83bbf8231dc5a588278646c7923b
Top:    6c14f92dfc0009489b694c8d1d7705c2ac18fabf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 02:03:29 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
